client/rest-high-level/src/test/java/org/elasticsearch/client/RequestTests.java

```diff
@@ -249,6 +250,78 @@ public void testExists() {
         getAndExistsTest(Request::exists, HttpHead.METHOD_NAME);
     }
 
+    public void testIndicesExist() {
+        String[] indices = randomIndicesNames(1, 10);
+        boolean flatSettings = randomBoolean();
+        boolean includeDefaults = randomBoolean();
+        boolean local = randomBoolean();
+        boolean humanReadable = randomBoolean();
+        boolean ignoreUnavailable = randomBoolean();
+        boolean allowNoIndices = randomBoolean();
+        boolean expandToOpenIndices = randomBoolean();
+        boolean expandToClosedIndices = randomBoolean();
```

Comment:
can you use setRandomIndicesOptions?

Reply:
also add setRandomLocal, setRandomFlatSettings etc. methods. Those are going to be useful.